vars/shared-Publications.var.yml

```diff
@@ -38,7 +38,7 @@ extensions:
       type: "git"
       description: "CKAN Extension for Queensland Government Publications theme"
       url: "https://github.com/qld-gov-au/ckanext-publications-qld-theme.git"
-      version: "1.4.0"
+      version: "1.4.1"
```

Comment:
can you also increment opendata theme for the no index part also.

Reply:
Needs review of https://github.com/qld-gov-au/ckanext-data-qld/pull/220 first to make the release.